MATLAB/README.md

```diff
@@ -3,60 +3,295 @@
-   * mission_time(positive int64), Mission time (ns)
  * actuators
-   * firing_start_times, Times since inital GPS week to start firing (s)
-   * real_thrust_vectors_body, Real thruster forces (N)
-   * centers_of_thrust_body, Center of thrust for each firing (m)
-   * firing_on_times, How long firings last (s)
+   * firing_start_times, Times since inital GPS week to start firing each thruster (s)
+   * thrust_vectors_body, each thruster's force vector (N)
+   * centers_of_thrust_body, Center of thrust of each truster (m)
+   * firing_on_times, How long each thruster firing lasts (s)
    * wheel_commanded_rate, Commanded x,y,z wheel rate (rad/s)
    * wheel_commanded_ramp, Commanded x,y,z wheel ramp (rad/s/s)
    * magrod_real_moment_body, Real magnetorquer moment (A*m^2)
+ * sensors
+   * gyro_bias, (rad/s)
+   * magnetometer_bias, (T)
+   * sunsensor_real_normals, (unitless)
+   * sunsensor_measured_normals,  (unitless)
+   * gps_bias,  (m,m/s)
+   * gps_time_till_lock, 
+       time till the GPS gets a lock, starts at XX min, then counts down and stays at 0
+       when the antenna is pointing towards the GPS constellation (s)
+   * cdgps_bias, (m,m/s)
+   * cdgps_time_till_lock,
+        time till the carrier-phase differential GPS(CDGPS) gets a lock, starts at XX min, then counts down and stays at 0
+        when the gps antenna is pointing towards the GPS constellation the piksi antenna is pointing at the other sattelite (s)
+    
+Inaddition to this core state, other info about the satellite is retrieved using
+the function get_truth which takes a string and the dynamics of a satellite as input.
+
+For example,
+get_truth('magnetic field body',main_state.follower.dynamics)
+returns the real magnetic field in the body frame.
+
+get_truth('magnetic field eci',main_state.follower.dynamics)
+returns the real magnetic field in the eci frame.
+
+get_truth('quat body eci',main_state.follower.dynamics)
+returns the quaternion that rotates vectors from eci to body
+
+get_truth('dcm body eci',main_state.follower.dynamics)
+returns the direction cosine matrix(DCM) that rotates vectors from eci to body
+
+get_truth('rate body eci body',main_state.follower.dynamics)
+returns the angular rate of the transform from eci to body in the body frame.
+
+get_truth('rate body',main_state.follower.dynamics)
+short hand for 'rate body eci body'
+returns the angular rate of the transform from eci to body in the body frame.
+
+get_truth('rate body eci eci',main_state.follower.dynamics)
+returns the angular rate of the transform from eci to body in the eci frame.
+
+get_truth('velocity eci',main_state.follower.dynamics)
+returns the velocity of the center of mass of the sat relative to earth in the eci frame.
+
+get_truth('velocity ecef',main_state.follower.dynamics)
+velocity measurements take into account additional velocity from rate cross radius.
+returns the velocity of the center of mass of the sat relative to earth in the ecef frame.
```

Comment:
I think this is great and super clean way of doing it. For some reason thought it'd be worse than this on the phone. Even if we can't call the functions directly from python or something we can just dumped them over all at one time.

Reply:
Most of the matlab functions should be callable from python. The matlab engine automatically converts python strings to matlab strings, and python dictionaries to matlab structures. [python matlab type conversions](https://www.mathworks.com/help/matlab/matlab_external/pass-data-to-matlab-from-python.html)